prinz/src/main/scala/pl/touk/nussknacker/prinz/model/Model.scala

```diff
@@ -1,14 +1,29 @@
 package pl.touk.nussknacker.prinz.model
 
-trait Model {
+import pl.touk.nussknacker.prinz.model.SignatureProvider.ProvideSignatureResult
 
-  def getName: ModelName
+trait Model {
 
-  def getVersion: ModelVersion
+  def getMetadata: ModelMetadata = ModelMetadata(getName, getVersion, getSignature, getParameterDefinition)
```

Comment:
I would opt for hiding internal signature in metadata pushed to the enricher if possible.

Reply:
will try to do it in #177